llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1176,8 +1180,11 @@ PassBuilder::buildModuleSimplificationPipeline(OptimizationLevel Level,
   // Enable contextual profiling instrumentation.
   const bool IsCtxProfGen = !IsPGOInstrGen && IsPreLink &&
                             PGOCtxProfLoweringPass::isContextualIRPGOEnabled();
+  const bool IsCtxProfUse = !UseCtxProfile.empty() && !PGOOpt &&
```

Comment:
Rather than checking`!PGOOpt` here and further below, should it be an error to combine these options?

Reply:
maybe, but I'm not seeing the usual error handling mechanism here; also, eventually I want to use PGOOpt (after some refactoring, because right now it's state is quite involved, see its ctor). While ctx_prof is in development, seems overkill to solve all these.